pkgs/applications/audio/ardour/default.nix

```diff
@@ -85,63 +90,71 @@ in stdenv.mkDerivation rec {
     fftw
     fftwSinglePrec
     flac
+    fluidsynth
     glibmm
     gtkmm2
+    hidapi
+    itstool
+    libarchive
     libjack2
     liblo
+    libltc
     libogg
+    libpulseaudio
+    librdf_raptor
+    librdf_rasqal
     libsamplerate
     libsigcxx
     libsndfile
     libusb1
-    fluidsynth_1
-    hidapi
-    libltc
-    qm-dsp
+    libuv
+    libwebsockets
     libxml2
+    libxslt
     lilv
     lrdf
     lv2
     pango
+    perl
```

Comment:
Why is `perl` needed?

Reply:
It's used in a few tools and in the build iiuc:  https://github.com/Ardour/ardour/search?q=perl&unscoped_q=perl